test/manual/amp-iframe-resize.amp.html

```diff
@@ -5,59 +5,53 @@
-  <script async src="../../dist/amp.js"></script>
 </head>
 <body>
-  The iframes in the initial viewport should not be resized automatically after 2 seconds.
+  <h1>Scroll down to see the iframe</h1>
+  <div style="height: 120vh"></div>
   <amp-iframe width=300 height=200
       sandbox="allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox"
       layout="responsive"
       frameborder="0"
       resizable
-      src="http://iframe.localhost:8000/test/manual/resize-fixture-iframe.html">
+      src="http://iframe.localhost:8000/iframe-echo-message">
       <div overflow tabindex=0 role=button aria-label="Read more">Read more!</div>
       <amp-img layout="fill" src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w400-h300-no-n" placeholder></amp-img>
   </amp-iframe>
-  <amp-iframe width=300 height=200
-        sandbox="allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox"
-        layout="responsive"
-        frameborder="0"
-        resizable
-        src="http://iframe.localhost:8000/test/manual/resize-fixture-iframe.html">
-        <div overflow tabindex=0 role=button aria-label="Read more">Read more!</div>
-        <amp-img layout="fill" src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w400-h300-no-n" placeholder></amp-img>
-    </amp-iframe>
-    <amp-iframe width=300 height=200
-        sandbox="allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox"
-        layout="responsive"
-        frameborder="0"
-        resizable
-        src="http://iframe.localhost:8000/test/manual/resize-fixture-iframe.html">
-        <div overflow tabindex=0 role=button aria-label="Read more">Read more!</div>
-        <amp-img layout="fill" src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w400-h300-no-n" placeholder></amp-img>
-    </amp-iframe>
-    <amp-iframe width=300 height=200
-    sandbox="allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox"
-        layout="responsive"
-        frameborder="0"
-        resizable
-        src="http://iframe.localhost:8000/test/manual/resize-fixture-iframe.html">
-        <div overflow tabindex=0 role=button aria-label="Read more">Read more!</div>
-        <amp-img layout="fill" src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w400-h300-no-n" placeholder></amp-img>
-    </amp-iframe>
+  <div style="height: 120vh"></div>
+
+  <div id="panel">
+    <label for="width">width</label><input id="width" type="number" value="200">
+    <label for="height">height</label><input id="height" type="nubmer" value="200">
+    <button onclick="requestResize()">Resize</button>
+  </div>
+
+  <script>
+    function requestResize() {
```

Comment:
Are these manual test pages supposed to be valid AMPs? I believe this script tag will make the page fail validation. Is that true?